spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe CommentsController, type: :controller do
+  describe "POST #create" do
+    before do
+      @project = FactoryGirl.create(:project)
+      sign_in(@project.user)
+    end
+    it 'Response code is 200' do
+      xhr :post, :create, comment: { polycomment_type: "project", polycomment_id: 3, issue: false, project_name: @project.name, body: "heads are cool" }
```

Comment:
I think we are eventually going to implement glitterposts too. acts_as_commentable will be useful.
I going to keep pushing to same PR. You can mark it in progress?

Since we are going in details and why not add [acts-as-taggable-on](https://github.com/mbleigh/acts-as-taggable-on) as well?